ledger-core/v2/application/functest/wallet_add_amount_test.go

```diff
@@ -16,6 +16,7 @@ import (
 
 // Creates wallet, calls /wallet/add_amount and checks it's response body.
 func TestWalletAddAmount(t *testing.T) {
+	t.Log("C4921	Пополнение кошелька")
```

Comment:
do we reeeally need russian names in repository?

Reply:
removed